### PR TITLE
Update server dir location for Docker

### PR DESCRIPTION
**Pull request by djfarrelly**
Removes the copying of the file and instead relies on the mounted volume from docker compose.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -23,8 +23,7 @@ RUN R -e "install.packages('shiny', repos='http://cran.rstudio.com/')" \
           && wget http://download3.rstudio.org/ubuntu-12.04/x86_64/shiny-server-1.3.0.403-amd64.deb \
           && gdebi --n shiny-server-1.3.0.403-amd64.deb \
           && rm shiny-server-1.3.0.403-amd64.deb \
-          && mkdir -p /srv/shiny-server \
-          && cp -R /usr/local/lib/R/site-library/shiny/examples/* /srv/shiny-server/.
```
Comment (djfarrelly) [inline Dockerfile]: After our volume was mounted in this folder so we shouldn't need the example copied there

File: Dockerfile [hunk 2]
```diff
@@ -63,7 +62,6 @@ EXPOSE 3838
 CMD ["/sbin/my_init"]
 
 
-RUN apt-get update
```
Comment (djfarrelly) [inline Dockerfile]: Duplicate

File: Dockerfile [hunk 3]
```diff
@@ -72,4 +70,3 @@ RUN R -e "install.packages(c('XML', 'ggplot2', 'downloader', 'data.table', 'dply
 COPY dependencies /srv/shiny-server/dependencies
 RUN R -e "install.packages('/srv/shiny-server/dependencies/rCharts/master.tar.gz', repos = NULL, type = 'source')"
 RUN R -e "library(rCharts)"
-COPY shiny-server /srv/shiny-server/
```
Comment (djfarrelly) [inline Dockerfile]: Hey @michael-erasmus! I removed this as I think it was confusing docker compose of the directory from being mounted in this same location. I am curious though as I think this copy is needed for building and deploying to AWS - do you have any insight into this one?

Comment (nieldlr) [inline Dockerfile]: Hi @djfarrelly @michael-erasmus,

I deployed this branch to the diversity-test environment and it seems to not have completely worked :) I'm not sure if I did things correctly here, but this might impact the deploy perhaps?

http://buffer-diversity-test.elasticbeanstalk.com/

Comment (djfarrelly) [inline Dockerfile]: Hey @nieldlr! I think we may need to revert this change I made to get it working for deploys. Maybe we can add a comment that if someone runs into issues building the Docker image locally they can comment that line out. As an alternative solution we could potentially configure AWS to mount that folder as a volume, but adding this line back in should get it done faster!

**Comment by djfarrelly** [comment]
Archiving this one.
